mkdocs/mkdocs.yml

```diff
@@ -173,6 +173,7 @@ nav:
       - Common Tasks: reference/tasks/common.md
       - Metrics Tasks: reference/tasks/metrics.md
       - Notification Tasks: reference/tasks/notification.md
+      - Dynamic Variable Substitution: reference/tasks/interpolation.md
```

Comment:
`mkdocs` does not feel like the the right place for this link. This link needs to be part of `tasks.md` file instead.

For starters, `interpolation` is not a separate task and is the exception in that list.

I actually anticipate other tasks (especially GitOps) tasks being written, where the interpolation behavior can differ from everything we have done so far.

Reply:
Moved as suggested. I hope that we will figure out how to extend/refactor the notion instead of having different concepts.